lib/ansible/modules/system/sysctl.py

```diff
@@ -211,12 +211,17 @@ def _values_is_equal(self, a, b):
     def _parse_value(self, value):
         if value is None:
             return ''
-        elif isinstance(value, bool):
+        if isinstance(value, bool):
```

Comment:
This change is not covered in the description of your PR.  Please amend the description to include the reasoning, or remove it from this PR.